Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -6507,3 +6507,27 @@ def search_indicators_by_version(self, from_date=None, query='', size=100, to_da
     @property
     def page(self):
         return self._page
+
+
+class AutoFocusKeyRetriever:
+    """AutoFocus API Key management class
+    :type api_key: ``str``
+    :param api_key: Auto Focus API key coming from the integration parameters
+    :type override_default_credentials: ``bool``
+    :param override_default_credentials: Whether to override the default credentials and use the
+     Cortex XSOAR given AutoFocus API Key
+    :return: No data returned
+    :rtype: ``None``
+    """
+    def __init__(self, api_key, override_default_credentials):
+        # demisto.getAutoFocusApiKey() is available from version 6.2.0
+        if not api_key:
+            if not is_demisto_version_ge("6.2.0"):  # AF API key is available from version 6.2.0
+                raise Exception('For versions earlier than 6.2.0, configure an API Key.')
```

Comment:
It's a bad practice to use `Exception`, use `DemistoException` or create a new exception like `CredentialsException`